diff_drive_controller/CMakeLists.txt

```diff
@@ -3,13 +3,14 @@ project(diff_drive_controller)
 
 find_package(catkin REQUIRED COMPONENTS
     controller_interface
-    urdf
+    nav_msgs
     realtime_tools
     tf
-    nav_msgs)
+    urdf
+)
 
 catkin_package(
-  INCLUDE_DIRS include
+  CATKIN_DEPENDS nav_msgs
```

Comment:
This was stupid, either should be all (following other packages) or nothing at all (following the "this is a plugin, therefore noone should link against it)